### PR TITLE
data-set

### PR DESCRIPTION
**Pull request by DieuCrevette**
NOT FINISHED DO **NOT CLOSE** @Maximellerbach

### CODE REVIEW

**Comment by DieuCrevette** [comment]
two erros on the test functions :

- [ ] Assert error on the test_load_data_set_generator
- [ ] Couldn't convert test___get_time_stamp (idk why need of investigation)

**Review by Maximellerbach — APPROVED** [review]
lgtm

File: autopylot/datasets/dataset.py [hunk 1]
```diff
@@ -50,4 +77,45 @@ def load_dataset_generator(dirpath):
...
+def load_multiple_sorted_dataset(dirpath):
+    """Load multiple sorted dataset
+
+    Args:
+        dirpath (string): path of directory which contains json and png.
+
+    Returns:
+       lsit[lsit[dict]]: sorted data.
+    """
+    return load_multiple_dataset(__sort_paths(dirpath))
```
Comment (Maximellerbach) [inline autopylot/datasets/dataset.py:102]: wait wait wait, this is not coherent, what are we sorting ?

**Review by Maximellerbach — CHANGES_REQUESTED** [review]

File: autopylot/datasets/dataset.py [hunk 2]
```diff
@@ -50,4 +77,45 @@ def load_dataset_generator(dirpath):
...
+
+
+def load_multiple_data_set_generator(dirpath, flat=False):
```
Comment (Maximellerbach) [inline autopylot/datasets/dataset.py:105]: typo here

**Comment by Maximellerbach** [comment]
The tests are not passing because of the fact that linux is not behaving in the same way regarding the file order, let's see this afternoon what we can do to solve this.
my guess would be to add a mark for the functions only working on windows, and disable them in the github action.

**Review by Maximellerbach — APPROVED** [review]
Good for merging !

**Pull request merged**